commons-mfa/src/main/java/org/exoplatform/mfa/filter/MfaFilter.java

```diff
@@ -44,7 +44,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
     MfaService mfaService = container.getComponentInstanceOfType(MfaService.class);
 
     String requestUri = httpServletRequest.getRequestURI();
-    if (httpServletRequest.getRemoteUser()!=null &&
+    if (mfaService.isMfaFeatureActivated() && httpServletRequest.getRemoteUser()!=null &&
```

Comment:
```suggestion
    if (httpServletRequest.getRemoteUser()!=null && mfaService.isMfaFeatureActivated() &&
```

Check first if we have a known user